### ISSUE TITLE
Some i.redd.it images doesn't hover

### ISSUE THREAD

**Issue by ahoibakk** [extesy/hoverzoom#289]
## Description of the problem
While browsing reddit, front page hovering some images hosted at i.redd.it doesn't 'hover'. If you open the link, and then go back, the image will hover.

## Url of the page where the problem occurs (not the image url)
From the front page, this [image](https://www.reddit.com/r/EarthPorn/comments/6tyu22/i_blah_blah_blah_at_the_break_of_dawn_blah_blah/) doesn't hover, however, browsing directly from the /r/earthporn, the image does hover.

## Steps to reproduce the problem (include image url here)
1.Go to reddit.com
2. Hover random i.redd.it images
3. Some doesn't hover
4. Click the miage
5. Go back
6. Image hovers

## OS version, extension version
Fully patched Windows 10, Chrome 60.0.3112.101 (Official Build) (64-bit), HoverZoom+ 1.0.123

**Comment by ahoibakk** [comment]
Another [link ](https://www.reddit.com/r/trashy/comments/6udgso/not_sure_if_this_is_trashy_or_fabulous/) that doesn't work.
Again, if I go the the trashy subreddit, it works fine, but not from /r/all.

**Comment by denaje** [comment]
I don't think so, but I'll keep an eye out for it. I think I have had some occasional issues with images not loading on hover, but I haven't paid attention to the pattern because it was so rare. If it happens again to me, I'll keep track.

**Comment by ragnarkarlsson** [comment]
I've noticed that gifv are a little hit and miss at times, but I haven't had any issues with jpg or png extensions nor standard animated gif.
